packages/web-components/src/components/tag/tag.scss

```diff
@@ -9,22 +9,93 @@ $css--plex: true !default;
 
 @use '@carbon/styles/scss/config' as *;
 @use '@carbon/styles/scss/utilities' as *;
+@use '@carbon/styles/scss/theme' as *;
+@use '@carbon/styles/scss/spacing' as *;
 @use '@carbon/styles/scss/breakpoint' as *;
+@use '@carbon/styles/scss/components/tooltip/index' as *;
 @use '@carbon/styles/scss/components/tag/index' as *;
+@use '@carbon/styles/scss/components/tag/mixins' as *;
 @use '@carbon/styles/scss/layout' as *;
 
-:host(#{$prefix}-tag) {
+:host(#{$prefix}-tag),
+:host(#{$prefix}-dismissible-tag) {
   @extend .#{$prefix}--tag;
+}
+
+:host(#{$prefix}-selectable-tag),
+:host(#{$prefix}-operational-tag) {
+  display: inline-block;
+  inline-size: fit-content;
+}
+
+:host(#{$prefix}-tag:not(:first-child)),
+:host(#{$prefix}-dismissible-tag:not(:first-child)),
+:host(#{$prefix}-selectable-tag:not(:first-child)) #{$prefix}-tag,
+:host(#{$prefix}-operational-tag:not(:first-child)) #{$prefix}-tag {
+  margin-inline-start: 0;
+}
+
+:host(#{$prefix}-tag),
+:host(#{$prefix}-dismissible-tag) {
+  border: 0;
+}
+
+:host(#{$prefix}-tag[has-custom-icon]),
+:host(#{$prefix}-dismissible-tag[has-custom-icon]) {
+  padding-inline-start: $spacing-02;
+}
+
+:host(#{$prefix}-selectable-tag) #{$prefix}-tag {
+  @extend .#{$prefix}--tag--selectable;
+}
+
+:host(#{$prefix}-operational-tag) #{$prefix}-tag {
+  @extend .#{$prefix}--tag--operational;
+}
+
+:host(#{$prefix}-selectable-tag[selected]) #{$prefix}-tag {
+  @extend .#{$prefix}--tag--selectable-selected;
+}
+
+:host(#{$prefix}-dismissible-tag) {
+  display: none;
+  padding-block: 0;
+  padding-inline-end: 0;
+}
+
+:host(#{$prefix}-dismissible-tag[open]) {
+  display: inline-flex;
+}
+
+:host(#{$prefix}-dismissible-tag) #{$prefix}-tooltip {
+  @extend .#{$prefix}--icon-tooltip;
+  @extend .#{$prefix}--tag-label-tooltip;
+}
+
+:host(#{$prefix}-dismissible-tag) .#{$prefix}--tag__close-icon {
+  @extend .#{$prefix}--tag__close-icon;
+}
+
+:host(#{$prefix}-dismissible-tag) ::slotted([slot='icon']) {
+  @extend .#{$prefix}--tag__custom-icon;
+}
+
+:host(#{$prefix}-tag),
+:host(#{$prefix}-tag-skeleton),
+:host(#{$prefix}-dismissible-tag) {
   @extend .#{$prefix}--layout--size-md;
 
   @include emit-layout-tokens();
   // tag overrides layout tokens
   --cds-layout-size-height-xs: 1.125rem;
   --cds-layout-size-height-sm: 1.125rem;
   --cds-layout-size-height-md: 1.5rem;
+  --cds-layout-size-height-lg: 2rem;
```

Comment:
```suggestion
  --cds-layout-size-height-md: $spacing-06;
  --cds-layout-size-height-lg: $spacing-07;
```

Reply:
Actually looks like I have to give it a hardcoded value here as the token doesn't get complied it looks like

<img width="477" alt="Screenshot 2025-03-26 at 1 23 04 PM" src="https://github.com/user-attachments/assets/54502118-6877-4f92-8dba-86d4d960d585" />